geospace-lib/src/main/scala/com/socrata/geospace/lib/Utils.scala

```diff
@@ -15,8 +15,8 @@ object Utils extends Logging {
   // Returns (FreeMemoryInMB, FreeMemoryPercentageInt (0-100))
   private def getFree: (Int, Int) = {
     val freeMB = Runtime.getRuntime.freeMemory / MB
-    val totalMB = Runtime.getRuntime.maxMemory / MB
-    val freePct = freeMB * Pct / totalMB
+    val total = Runtime.getRuntime.maxMemory
+    val freePct = freeMB * Pct * MB / total // not converting total now is less expensive
```

Comment:
Juggling one piece of the equation from line 18 to 19 is no more or less expensive. It would be less expensive to defer unit conversion until the end.
```
val free = Runtime.getRuntime.freeMemory
val total = Runtime.getRuntime.maxMemory
val freePct = free * Pct / total
val freeMB = free / MB
```